packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -274,6 +275,9 @@ class _TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
       _controller.forward();
       return false; // Already visible.
     }
+    if (context == null) {
```

Comment:
you can use `if (mounted)` instead, but as I said in my other comment, I don't fully understand yet why this would be necessary here.

Reply:
actually the problem may be we are missing a 
`_showTimer.cancel()` elsewhere int his code when the unmount event occurs.